corehq/apps/app_manager/suite_xml.py

```diff
@@ -1922,11 +1923,25 @@ def get_datum_meta_module(self, module, use_filter=False):
                     detail_inline = bool(detail.pull_down_tile)
                     break
 
+            fixture_select_filter = ''
+            if datum['module'].fixture_select.active:
+                datums.append({
+                    'datum': SessionDatum(
+                        id='fixture_value',
+                        nodeset="instance('item-list:{ft}')/{ft}_list/{ft}"
+                            .format(ft=datum['module'].fixture_select.fixture_type),
```

Comment:
You can use `ItemListFixtureXpath(module.fixture_select.fixture_type).instance()`